code/modules/clothing/under/jobs/security.dm

```diff
@@ -9,7 +9,7 @@
  * Security
  */
 /obj/item/clothing/under/rank/warden
-	desc = "It's made of a slightly sturdier material than standard jumpsuits, to allow for more robust protection. It has the word \"Warden\" written on the shoulders."
+	desc = "A jumpsuit made of strong material, providing better protection. It has the word \"WARDEN\" written on the shoulders."
```

Comment:
all the ones that say `providing better protection` should probably say `providing it better protection`